docs/content/opa-1.md

```diff
@@ -0,0 +1,201 @@
+---
+title: OPA 1.0
+kind: misc
+weight: 4
+---
+
+OPA v1.0 will introduce breaking changes to the Rego language. 
+Here, these changes are described, along with the presentation of tools to make new and existing policies compatible with OPA 1.0.
+These changes are backwards compatible in the sense that it is possible to author policies that are compatible with both OPA v1.0 and older versions of OPA.
+
+## The `future.keywords` imports
+
+The `in`, `every`, `if` and `contains` keywords have been introduced over time, and currently require opt-in to prevent them from breaking policies that existed before their introduction. 
+The `future.keywords` imports facilitate this opt-in mechanism.
+These keywords help to increase the readability of policies and provide syntactic sugar for commonly used operations such as iteration, membership checks, defining multi-value rules, and so on.
+There is growing adoption of these keywords and their usage is prevalent in the OPA documentation, Rego Playground, etc.
+
+In OPA v1.0 the `in`, `every`, `if` and `contains` keywords will be part of the language by default and the `future.keywords` imports will become a no-op.
+A policy that makes use of these keywords, but doesn't import `future.keywords` is valid in OPA v1.0 but not in older versions of OPA.
+
+### Making new and existing policies compatible with OPA v1.0
+
+1. A new `rego.v1` import has been introduced that, when used, makes OPA apply all restrictions that will eventually be enforced by default in OPA v1.0. 
+   If a Rego module imports `rego.v1`, it means applicable `future.keywords` imports are implied. It is illegal to import both `rego.v1` and `future.keywords` in the same module.
+2. The `--rego-v1` flag on the `opa fmt` command will rewrite existing modules to use the `rego.v1` import instead of `future.keywords` imports.
+3. The `--rego-v1` flag on the `opa check` command will check that either the `rego.v1` import or applicable `future.keywords` imports are present if any of the `in`, `every`, `if` and `contains` keywords are used in a module.
+
+### Backwards compatibility in OPA v1.0
+
+To avoid breaking existing policies and give users a chance to update them such that these future keywords don’t clash with, for example, existing variable names, a new flag called `--v0-compatible` will be added to OPA commands such as `opa run` to maintain backward compatibility. 
+OPA’s Go SDK and Go API will be equipped with similar functionality, as well. 
+In addition to this, the bundle manifest will also be updated to include a bit that indicates whether pre-OPA v1.0 behavior is desired. 
+OPA tooling such as the `opa build` command will be equipped with a new flag that will allow users to control the relevant bit in the manifest. 
+This should be useful for cases when pre-OPA v1.0 functionality is desired but the users themselves have no control over the OPA deployment and hence cannot set the CLI flag.
```

Comment:
I think this applies to 

> In addition to this, the bundle manifest will also be updated to include a bit that indicates whether pre-OPA v1.0 behavior is desired.

rather than

> OPA tooling such as the `opa build` command will be equipped with a new flag that will allow users to control the relevant bit in the manifest.